### DISCUSSION TITLE
[Documentation SIG] Organizing

### DISCUSSION THREAD

**Discussion by ryw** [apache/airflow#11949] [General]
_Moving this discussion from slack #documentation_

## Objective

* Develop a shared future vision for Airflow docs
* Continuously improve Airflow docs

## Process

* Keep it Agile
* Call every 2 weeks: [Google Calendar Link](https://calendar.google.com/calendar/embed?src=c_03cnamrtfr3n7nmhdka3i2j94g%40group.calendar.google.com&ctz=America%2FNew_York)
* Work managed via a Github project board: [Github Project Link](https://github.com/apache/airflow/projects/6)
* Small team willing to put time in to make our docs better

## Activities

* Get stats on Airflow’s documentation usage, hits, which pages are commonly visited (GA Analytics, @mik-laj has access)

## Volunteers

| Name | Github Handle | Role |
| --- | --- | --- |
| Ry Walker | @ryw | Organizer |
| Aditya Trivedi | @iadi7ya | |
| Francesco Mucio | @francescomucio | |
| Jacob Ward | @jward-bw | |
| James Timmins | @jhtimmins | |
| Kamil Breguła | @mik-laj | |
| Kaxil Naik | @kaxil | |
| Paola Peraza Calderon | @paolaperaza | |
| Patrick Cando | @pcandoalmeida | |
| Xinbin Huang | @xinbinhuang | |

## Examples of good docs

* [Rails Guides](https://guides.rubyonrails.org/)
* [Terraform](https://www.terraform.io/docs/index.html)
* [TensorFlow](https://www.tensorflow.org/api_docs)
* [Kubeflow](https://www.kubeflow.org/docs/)
* [PostgreSQL](https://www.postgresql.org/docs/13/index.html)
* [PyTorch](https://pytorch.org/docs/stable/index.html)

## Convenient commands

`./breeze build-docs`

## Related links

* https://docs.google.com/spreadsheets/d/1vIqU6DPYJl0gMsc2uWEakNyP2hkhulOQMrpujrKC0B4/edit#gid=0
* https://github.com/apache/airflow/issues/11423
* https://github.com/apache/airflow/issues/11152
* [Email thread](https://lists.apache.org/thread.html/rc6331d0bf97d91dc88853c992513f4e886f113c1cff030876996859e%40%3Cdev.airflow.apache.org%3E)
* https://github.com/apache/airflow/issues/11077
* [new api docs](https://airflow.readthedocs.io/en/latest/stable-rest-api-ref.html)
* [getting latest docs on the website](https://github.com/apache/airflow-site/issues/270)

**Comment by mik-laj** [comment]
> Get stats on Airflow’s documentation usage, hits, which pages are commonly visited (GA Analytics)

I have access to GA. Please assign it to me.

**Reply by kaxil** [reply]
I have access to GA too, will be present in the next call
